tox.ini

```diff
@@ -21,6 +21,8 @@ basepython = python3
 [testenv]
 usedevelop = true
 passenv = DJANGO_SETTINGS_MODULE PYTHONPATH HOME DISPLAY
+setenv =
+    PYTHONDONTWRITEBYTECODE = 1
```

Comment:
Agree it looks funny, but all of the docs and examples show usage with a space on either side of the `=`. I'll test it both ways and if it works without the spaces I'll change to that.